scripts/generate-docs-checkout.ts

```diff
@@ -27,8 +27,8 @@ const postPurchase = {
 };
 
 const componentsPageContent = (url: string) => ({
-  title: 'Components for checkout extensions',
-  frontMatterDescription: 'A list of components for checkout extensions.',
+  title: 'Components for Post-purchase extensions',
```

Comment:
Is this title not supposed to be generic Checkout UI extensions?

It looks like the post purchase

Reply:
For unite we want to scope it only to post-purchase. Afterward, we'll change it back